topics/admin/tutorials/tool-management/slides.html

```diff
@@ -351,55 +694,32 @@
 └── varscan_somatic.xml                # wrapper for tool #3
 ```
 
+???
+
+This is a more complex toolshed repository containing three tool wrappers related
+to the same software.  Installing this repository will install three tools into
+the tool panel.
+
 ---
 
 ## Data tables and loc files
 
 - Several tools need to access **reference data**, e.g. genome sequences or indexes for aligner
 - A *loc file* is a tab-separated file containing metadata and paths for a set of reference data
 - A *data table* describes the columns of a loc file used by a tool
+- Installing CVMFS provides access to a large amount of reference data.  There are also
+  data manager tools for installing reference data.
 - You can read more about this in the [Reference Genomes in Galaxy slides](../reference-genomes/slides.html)
-
----
-
-## Suite repositories
-
-- Suites can be used to split up a set of tools into multiple repositories
-  - A suite is a single repository that 'depends' on many others
-  - When you install the suite, all 'dependency repositories' will be installed too
-- With a tweak to `.shed.yml`, [Planemo](https://planemo.readthedocs.io) can upload a directory of tools to the TS as a suite of separate tools
```

Comment:
Not sure where to put it, but I'd keep the slide on Suites.

Reply:
I'll put it back where it was (after the slide on reference data).  It fits into the 'More on Tool Management' slides